derive_builder_core/src/macro_options/darling_opts.rs

```diff
@@ -245,6 +245,9 @@ pub struct Field {
     default: Option<DefaultExpression>,
     #[darling(default)]
     try_setter: Flag,
+    /// Custom builder field type and build method
+    #[darling(default)]
+    custom: Option<CustomField>,
```

Comment:
Could this go in `field` instead?

```rust
#[builder(field(private, type = "...", build = "..."))]
```

That feels pretty nice.

Reply:
Yes, it turns out.